### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA announces additional inappropriate information when "Azure sign in" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1476]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3.  "Azure Sign in" dialog, will open.
4. Observe Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Azure sign in" dialog   is opened 

**Jaws behaviour:**
 Azure Sign In dialog
An authentication file with credentials for an Azure Active Directory service principal will be used for automated sign ins. You will manually sign in using your Azure credentials as needed.
Sign in Button
To activate press spacebar.

**NVDA Behaviour:**
Azure Sign In  dialog  Authentication file:
Sign in  button

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
Ie .  Azure Sign In dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![azure sign in](https://user-images.githubusercontent.com/22956130/36104721-c99418ac-1038-11e8-88ff-a212ca2305c4.jpg)
![azure signin-nvda](https://user-images.githubusercontent.com/22956130/36104726-ccc9571c-1038-11e8-8413-7a56a4516e24.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749915
